frontend/src/app/spot/components/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,72 @@
+<ng-container
+  *ngFor="let crumb of content.crumbs; let i = index; let last = last; let first = first"
+>
+  <!-- Desktop crumbs -->
+  <div
+    *ngIf="(first && !last) || i === content.crumbs.length - 2 || i === content.crumbs.length - 3 "
+    class="spot-breadcrumbs--crumb hidden-for-mobile"
+    [ngClass]="{'spot-breadcrumbs--crumb_shrink': !first }"
+  >
+    <button
+      type="button"
+      class="spot-link spot-breadcrumbs--crumb-action-item"
+      [title]="crumb.text"
```

Comment:
This should have some kijnd of click listener, no?

Reply:
true, currently there is no functionality attached yet, but the click handler can get added.